app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -33,6 +33,25 @@ message Target {
     // Name of the scheduler to trigger this target.
     // Defaults to being triggered by cocoon.
     optional SchedulerSystem scheduler = 8 [default = cocoon];
+    optional PhaseConfig presubmit = 9;
+    optional PhaseConfig postsubmit = 10;
+}
+
+// CI Phase (i.e. pre-submit vs post-submit).
+// Next ID: 3
+message PhaseConfig {
+    optional Pool pool = 1;
```

Comment:
Is there a use case for pool being different on post submit and presubmit runs?

Reply:
pre-submit would be try, and post-submit prod, right?

Reply:
Yes, but what if a target has `bringup=true` we schedule those builds on staging? Then we wouldn't need target to know about the luci pools.